tests/test_custom_vocabularies_client.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import re
+import json
+from requests.exceptions import HTTPError
+from src.rev_ai.apiclient import RevAiAPIClient
+from src.rev_ai import __version__
+from src.rev_ai.custom_vocabularies_client import RevAiCustomVocabulariesClient
+from tests.helpers.errors import get_error_test_cases
+
+TOKEN = "token"
+
+
+class TestCustomVocabulariesClient:
+    def test_constructor_with_success(self):
+        client = RevAiCustomVocabulariesClient(TOKEN)
+
+        headers = client.default_headers
+
+        assert headers.get(
+            'User-Agent') == 'RevAi-PythonSDK/{}'.format(__version__)
+        assert headers.get('Authorization') == 'Bearer {}'.format(TOKEN)
+
+    @pytest.mark.parametrize('token', [None, ''])
+    def test_constructor_with_no_token(self, token):
```

Comment:
We should have tests for each request to make sure they call `BaseClient._make_http_request` with the proper arguments

Reply:
This might help
https://stackoverflow.com/questions/12219967/how-to-mock-a-base-class-with-python-mock-library

Reply:
There is a follow up ticket for writing tests.